mac/Keyman4MacIM/Keyman4MacIM/KMInputController.m

```diff
@@ -32,95 +67,391 @@ - (id)initWithServer:(IMKServer *)server delegate:(id)delegate client:(id)inputC
 }
 
 - (NSUInteger)recognizedEvents:(id)sender {
-    return (NSKeyDownMask | NSLeftMouseDownMask);
+    return (NSKeyDownMask | NSLeftMouseDownMask | NSLeftMouseUpMask);
 }
 
 - (BOOL)handleEvent:(NSEvent *)event client:(id)sender {
     if ([self.AppDelegate debugMode])
         NSLog(@"Event = %@", event);
     
-    if (event == nil || sender == nil)
-        return NO;
+    if (event == nil || sender == nil || self.kmx == nil)
+        return NO; // Not sure this can ever happen.
     
     // OSK key feedback from hardware keyboard is disabled
     /*if (event.type == NSKeyDown)
         [self.AppDelegate handleKeyEvent:event];*/
     
-    if (self.kmx == nil)
+    if (event.type == NSLeftMouseDown || event.type == NSLeftMouseUp ) {
```

Comment:
Do we want to do the same for other mouse buttons?

Reply:
What other buttons?! It's a Mac :-) I'll look into this...

Reply:
Sadly many people use Macs with Logitech and other mice that have at least two buttons ;-)